src/sources/fluent/mod.rs

```diff
@@ -162,97 +159,95 @@ impl From<decode::Error> for DecodeError {
 }
 
 #[derive(Debug)]
-struct FluentDecoder {
-    // unread frames from previous fluent message
-    unread_frames: VecDeque<(FluentFrame, usize)>,
-}
+struct FluentDecoder {}
```

Comment:
Nit, if this has no fields, could we just do `struct FluentDecoder;`?

Reply:
Sure, it originally had 1 field but I removed it and didn't bother removing the `{}`.